modules/docs/src/Volo.Docs.Web/Pages/Documents/Project/Index.cshtml.cs

```diff
@@ -772,7 +781,9 @@ private async Task SetDocumentPreferencesAsync()
                 return;
             }
 
-            var availableParameters = await _documentSectionRenderer.GetAvailableParametersAsync(Document.Content);
+            var availableParameters = await _webDocumentSectionRenderer.GetAvailableParametersAsync(Document.Content);
+            var parameterCombinations = new List<Dictionary<string, string>>();
```

Comment:
What is this variable used for?

Reply:
This is a test code, it should be deleted